packages/matchbox/src/components/Tooltip/Tooltip.js

```diff
@@ -1,124 +1,166 @@
-      horizontalOffset,
+      // These two props are broken
+      right, // Omitting as this is a valid system prop
+      bottom, // Omitting as this is a valid system prop
       forcePosition,
       disabled,
-      width = '210px'
-    } = this.props;
+      width = '13rem',
+      ...rest
+    } = props;
 
+    const systemProps = pick(rest);
     const positionTop = preferredDirection.top || (top && !forcePosition);
     const positionLeft = preferredDirection.left || (left && !forcePosition);
 
-    const wrapperClasses = classnames(
-      !disabled && this.state.hover && styles.hover,
-      dark && styles.dark,
-      positionTop && styles.top,
-      positionLeft && styles.left,
-    );
-
-    const offset = positionLeft ? { right: horizontalOffset } : { left: horizontalOffset };
+    const contentStyles = {
+      ...systemProps,
+      ...(dark ? { bg: 'gray.1000' } : {}), // TODO deprecate in favor of system props
+    };
 
     return (
-      <span className={wrapperClasses}>
-        <span className={styles.Tooltip} style={{ ...offset, width }}>
-          <span className={styles.Tip} />
-          <div className={styles.Content}>{content}</div>
-        </span>
-      </span>
+      <StyledTooltipContainer
+        display="block"
+        position="absolute"
+        top={positionTop ? 'auto' : '100%'}
+        bottom={positionTop ? '100%' : 'auto'}
+        left={positionLeft ? 'auto' : '0'}
+        right={positionLeft ? '0' : 'auto'}
+        // Tooltip gap to activator
+        mt={positionTop ? '0' : '100'}
+        mb={positionTop ? '100' : '0'}
+        visible={!disabled && hover}
+      >
+        <StyledContent
+          position="relative"
+          fontSize="200"
+          lineHeight="200"
+          p="300"
+          borderRadius="100"
+          bg="gray.1000"
+          color="white"
+          textAlign="left"
+          width={width}
+          zIndex="1"
+          {...contentStyles}
+        >
+          <Box overflow="hidden" maxHeight="11rem">
+            {content}
+          </Box>
+        </StyledContent>
+      </StyledTooltipContainer>
     );
   }
 
-  renderActivator = ({ activatorRef }) => (
-    <span
-      className={styles.Activator}
-      onMouseOver={this.handleMouseOver}
-      onMouseOut={this.handleMouseOut}
-      ref={activatorRef}>
-      {this.props.children}
-    </span>
-  )
-
-  render() {
+  function renderActivator({ activatorRef }) {
     return (
-      <TooltipOverlay
-        eventDebounce={this.props.eventDebounce}
-        portalId={this.props.portalId}
-        renderTooltip={this.renderTooltip}
-        renderActivator={this.renderActivator} />
+      <Box
+        display="inline-block"
+        onMouseOver={handleMouseOver}
+        onMouseOut={handleMouseOut}
+        ref={activatorRef}
+      >
+        {props.children}
+      </Box>
     );
   }
+
+  return (
+    <TooltipOverlay
+      eventDebounce={props.eventDebounce}
+      portalId={props.portalId}
```

Comment:
I thought there is an accessibility need to render the tooltip DOM with/alongside the button?

<img width="1390" alt="Screen Shot 2020-03-18 at 11 15 51 AM" src="https://user-images.githubusercontent.com/1335605/76976111-e53fb080-6909-11ea-98eb-19fcd8d8391b.png">